documentation/homework/week4/src/App.js

```diff
@@ -0,0 +1,25 @@
+// Import Initial Requirments
+import React, { Component } from 'react'
+import './App.css'
+
+// Import Provider
+import { Provider } from 'mobx-react'
+
+// Import Stores Index
+import Stores from './stores'
+
+// Import Components Index
+import Index from './components'
```

Comment:
You should call this something more descriptive. A component should never be called Index.